modules/go-setting-environment-annotations.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * /gitops/health-information-for-resources-deployment.adoc
+
+:_content-type: REFERENCE
+[id='go-setting-environment-annotations_{context}']             
+= Setting up environment labels and annnotations
+
+[discrete]
+== Environment labels:
+
+The environment application manifest should include `openshift.gitops/environment` label, and `destination.namespace` set. The set value for both the environment name and destination namespace should be the same.
+
+.Syntax
+[source,yaml]
+----
+spec:
+  labels:
+    openshift.gitops/environment: <environment-name>
+  destination:
+    namespace: <environment-name>
+----   
+
+.Example:
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: dev-env
+  namespace: openshift-gitops
+spec:
+  labels:
+    openshift.gitops/environment: dev-env
+  destination:
+    namespace: dev-env
+...
+----
+
+[discrete]
+== Environment annotations:
+The environment namespace manifest has to have `annotations.app.openshift.io/vcs-uri` and `annotations.app.openshift.io/vcs-ref` to specify the version controller code source of the application. The namespace name has to match the environment name.
```

Comment:
```suggestion
The environment namespace manifest must contain identical values for the `annotations.app.openshift.io/vcs-uri` field and the `annotations.app.openshift.io/vcs-ref` field that specify the version controller code source for the application. The namespace name must match the environment name.
```
- Per the ISG, use `must` rather than `have to` for mandatory actions. See the "have to" entry here: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#h